src/internal/m365/collection/drive/helper_test.go

```diff
@@ -59,3 +77,768 @@ func loadTestService(t *testing.T) *oneDriveService {
+			odConsts.DriveFolderPrefixBuilder(idx(drive, driveID)).String(),
+		},
+		elems...)...)
+}
+
+func parentDir(elems ...string) string {
+	return toPath(append(
+		[]string{odConsts.DriveFolderPrefixBuilder(id(drive)).String()},
+		elems...)...)
+}
+
+func driveParentDir(driveID any, elems ...string) string {
+	return toPath(append(
+		[]string{odConsts.DriveFolderPrefixBuilder(idx(drive, driveID)).String()},
+		elems...)...)
+}
+
+// just for readability
+const (
+	doMergeItems    = true
+	doNotMergeItems = false
+)
+
+// common item names
+const (
+	bar       = "bar"
+	delta     = "delta_url"
+	drive     = "drive"
+	fanny     = "fanny"
+	file      = "file"
+	folder    = "folder"
+	foo       = "foo"
+	item      = "item"
+	malware   = "malware"
+	nav       = "nav"
+	pkg       = "package"
+	rootID    = odConsts.RootID
+	rootName  = odConsts.RootPathDir
+	subfolder = "subfolder"
+	tenant    = "t"
+	user      = "u"
+)
+
+var anyFolderScope = (&selectors.OneDriveBackup{}).Folders(selectors.Any())[0]
+
+type failingColl struct{}
+
+func (f failingColl) Items(ctx context.Context, errs *fault.Bus) <-chan data.Item {
+	ic := make(chan data.Item)
+	defer close(ic)
+
+	errs.AddRecoverable(ctx, assert.AnError)
+
+	return ic
+}
+func (f failingColl) FullPath() path.Path                                        { return nil }
+func (f failingColl) FetchItemByName(context.Context, string) (data.Item, error) { return nil, nil }
+
+func makeExcludeMap(files ...string) map[string]struct{} {
+	delList := map[string]struct{}{}
+	for _, file := range files {
+		delList[file+metadata.DataFileSuffix] = struct{}{}
+		delList[file+metadata.MetaFileSuffix] = struct{}{}
+	}
+
+	return delList
+}
+
+// ---------------------------------------------------------------------------
+// limiter
+// ---------------------------------------------------------------------------
+
+func minimumLimitOpts() control.Options {
+	minLimitOpts := control.DefaultOptions()
+	minLimitOpts.PreviewLimits.Enabled = true
+	minLimitOpts.PreviewLimits.MaxBytes = 1
+	minLimitOpts.PreviewLimits.MaxContainers = 1
+	minLimitOpts.PreviewLimits.MaxItems = 1
+	minLimitOpts.PreviewLimits.MaxItemsPerContainer = 1
+	minLimitOpts.PreviewLimits.MaxPages = 1
+
+	return minLimitOpts
+}
+
+// ---------------------------------------------------------------------------
+// enumerators
+// ---------------------------------------------------------------------------
+
+func collWithMBH(mbh BackupHandler) *Collections {
+	return NewCollections(
+		mbh,
+		tenant,
+		idname.NewProvider(user, user),
+		func(*support.ControllerOperationStatus) {},
+		control.Options{ToggleFeatures: control.Toggles{
+			UseDeltaTree: true,
+		}},
+		count.New())
+}
+
+func collWithMBHAndOpts(
+	mbh BackupHandler,
+	opts control.Options,
+) *Collections {
+	return NewCollections(
+		mbh,
+		tenant,
+		idname.NewProvider(user, user),
+		func(*support.ControllerOperationStatus) {},
+		opts,
+		count.New())
+}
+
+// func fullOrPrevPath(
```

Comment:
Just checking, but what's the plan for this and other commented out code in the file?

Reply:
they get updated here: https://github.com/alcionai/corso/pull/4795/files#diff-48bc25f85f8c885368934862a2813c6fbd7dfb71f309f91169f605230a85895eR656